drv/i2c-devices/src/adm1272.rs

```diff
@@ -85,7 +85,7 @@ enum Trace {
     None,
 }
 
-ringbuf!(Trace, 32, Trace::None);
+ringbuf!(Trace, 8, Trace::None);
```

Comment:
Is this change intentional?

Reply:
Yeah -- that ring buffer was **way** oversized -- and I want to pay for a similar ring buffer in the lm5066 code. I have sized them both at a much more rational size, so it's net-minus on RAM usage.